libcxx/docs/index.rst

```diff
@@ -67,6 +67,10 @@ Getting Started with libc++
 Current Status
 ==============
 
+.. image:: https://github.com/llvm/llvm-project/actions/workflows/libcxx-build-and-test.yaml/badge.svg?branch=main&event=schedule
```

Comment:
IMO this is a really bad place to put this. I'm pretty sure most people don't care whether our CI is currently green, and this has _nothing_ to do with the text below.

Reply:
I have to agree to some extent. This is user documentation, not libc++ developer documentation. Also, we publish this page and freeze it for releases, but this link is relative so for example if you look at the page for the LLVM 18 release, it will contain this badge which may go green/red depending on the weather. This seems undesirable.

Instead, I think a better place to put this badge would be on the LLVM monorepo README which we can see at a glance by just looking at the repo -- other projects could consider doing the same thing and we'd have some kind of table that shows the status of all projects.

Reply:
Cool. I'll send a change. But I would like this on the website somewhere. I can put it down next to the builders.

Reply:
Just curious, but why not put it in the main LLVM README? Then the status is directly visible from the main GitHub page. This is the typical location for badges.